.gitignore

```diff
@@ -15,6 +15,7 @@ pip-wheel-metadata/UNKNOWN.dist-info/METADATA
 .vscode/settings.json
 .DS_Store
 .venv
+node_modules/
```

Comment:
We don't expect to have node_modules in the root folder. We may have it only in `ui` folder and it's listed in gitignore there.
The only case when node_modules appears in the root folder when you mistakenly run `yarn` on the root folder

Reply:
Okay, I'll revert this change.